package.json

```diff
@@ -2,10 +2,20 @@
   "name": "cross-domain-utils",
   "version": "1.0.9",
   "description": "Javascript module template.",
-  "main": "index.js",
+  "main": "dist/cross-domain-utils.js",
   "scripts": {
-    "test": "gulp test",
-    "postinstall": "node postinstall.js"
+    "postinstall": "flow-typed install",
+    "build": "babel-node ./node_modules/.bin/webpack --progress --output-path dist",
+    "karma": "babel-node ./node_modules/.bin/karma start",
```

Comment:
Yeah, would still like it. Often find myself with a ton of lint errors that I want to fix later, after I figure out the more important type errors.